### ISSUE TITLE
Favicons for bookmarks.

### ISSUE THREAD

**Issue by streetwolf** [sefeng211/faviconSwitcher#7]
From your release notes..."It can't modify the favicons for bookmarks, there's an existing bug on bugzilla about it."  I'm on Nightly and yes indeed it modifies bookmark favicons on my bookmark toolbar.  However, these modified bookmarks didn't have any favicons.  Does your statement refer to changing existing favicons?  I haven't tried that yet.  Really no need to.

**Comment by sefeng211** [comment]
Yup, that is exactly what I meant. Can't modified the existing favicons because the bookmark api doesn't allow us to manipulate the favicon.

**Comment by streetwolf** [comment]
Thought this might be the case.  The main reason for using FS in my case is to supply icons for missing favicons.  If a site has one there is really no reason to change it unless it makes no sense for the site or is just plain UGLY.  Mozilla is really intent on locking down Firefox for customization.  I just got ahold of some css code to reduce the gigantic size of the MegaBar in Nightly using userChrome.css.  That is until userChrome.css goes the way of the dinosaur.  Hmmm.. Can’t really say that anymore since scientists tell us birds are actually dinosaurs. 😊

------------
Regards,
Gary

**Comment by jhlee2222** [comment]
Has anyone figured out how to go about changing the bookmark favicons, or wiping them so FS can change them? I've gone so far as to try making manual changes to the favicons.sqlite file in my profile, but not able to make any progress.

**Comment by cm-fy** [comment]
> Has anyone figured out how to go about changing the bookmark favicons, or wiping them so FS can change them? I've gone so far as to try making manual changes to the favicons.sqlite file in my profile, but not able to make any progress.

I've tried editing the sqlite too, no good on waterfox 6. In pre-quantum ff, add-ons could allow even change the icons for bookmark folders, I miss that as well.
